assets/less/base/common.less

```diff
@@ -0,0 +1,431 @@
+    font-weight: 700;
+    text-align: center;
+    text-transform: uppercase;
+    margin-bottom: 10px;
+  }
+  p {
+    font-size: 14px;
+    color: @color-1;
+    font-family: @font-1;
+    text-align: center;
+  }
+}
+
+.team {
+  padding: 100px 0;
+}
+
+.team__items {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  align-items: stretch;
+}
+
+.team__item {
+  flex-basis: 22.5%;
+}
+
+.item__ava {
+  img {
+    object-position: center;
+    object-fit: cover;
+    max-width: 100%;
+  }
+}
+
+.item__name {
+  font-size: 22px;
+  color: @color-3;
+  font-family: @font-1;
+  padding-top: 30px;
+}
+
+.item__position {
+  font-size: 13px;
+  color: @color-4;
+  font-family: @font-1;
+  padding-top: 15px;
+  text-transform: uppercase;
+}
+
+.item__about {
+  font-size: 14px;
+  line-height: 1.5;
+  color: @color-5;
+  font-family: @font-1;
+  padding-top: 30px;
+}
+
+.item__socials {
+  padding-top: 30px;
+}
+
+.ico {
+  font-size: 20px;
+  color: @color-6;
+  width: 40px;
+  height: 40px;
+  border: 1px solid @color-6;
+  border-radius: 1px;
+  text-align: center;
+  padding-top: 10px;
+  transition-duration: 0.3s;
+  &:hover {
+    color: @color-4;
+    border: 1px solid @color-4;
+  }
+}
+
+.clients {
+  display: flex;
+  max-height: 500px;
+}
+
+.clients__photo {
+  flex-basis: 50%;
+
+  img {
+    object-position: center;
+    object-fit: cover;
+    width: 100%;
+    height: 100%;
+  }
+}
+
+.clients__review {
+  flex-basis: 50%;
+  background-color: #53354a;
+  padding: 6vw 21vw 11vw 4.5vw;
+  // padding: 100px 180px 100px 80px;
+}
+
+.clients__content {
+  font-size: 36px;
+  line-height: 1.2;
+  color: @color-2;
+  font-family: @font-3;
+  font-weight: 400;
+  font-style: italic;
+  padding-bottom: 30px;
+}
+
+.clients__subtitle {
+  font-size: 13px;
+  line-height: 1.6;
+  color: @color-2;
+  font-family: @font-1;
+  font-weight: 700;
+  padding-bottom: 50px;
+  text-transform: uppercase;
+}
+
+.clients__slider-wrap {
+  width: 115px;
+  margin: 0 auto;
+  div {
+    display: inline-block;
+    background-color: @color-1;
+    width: 13px;
+    height: 13px;
+    border-radius: 50%;
+    margin-right: 8px;
+    &:nth-child(2) {
+      background-color: @color-2;
+    }
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
+
+.download__inner {
+  padding: 100px 0;
+}
+
+.download__title {
+  font-size: 28px;
+  line-height: 0.75;
+  color: @color-3;
+  font-family: @font-1;
+  font-weight: 300;
+  text-align: center;
+  padding-bottom: 20px;
+}
+
+.download__subtitle {
+  font-size: 13px;
+  line-height: 1.6;
+  color: @color-3;
+  font-family: @font-1;
+  font-weight: 700;
+  text-align: center;
+  padding-bottom: 40px;
+  text-transform: uppercase;
+}
+
+.download__button {
+  padding-left: 80px;
+  padding-right: 80px;
+}
+
+.footer__contact-wrap {
+  background-color: #3a3e64;
+  padding: 70px 0;
+}
+
+.footer__contact {
+  display: flex;
+  justify-content: space-around;
+  & > div {
```

Comment:
& is redundant

Reply:
Why? I need select just first-inserted divs